pkg/daemon/daemon.go

```diff
@@ -333,6 +335,11 @@ func (d *Daemon) sharedErofsUmount(rafs *Rafs) error {
 		return errors.Wrapf(err, "umount erofs %s mountpoint, %s", err, mountpoint)
 	}
 
+	// delete fscache bootstrap cache file
+	if err := c.UnbindBlob("", fscaheID); err != nil {
```

Comment:
Please also comment the reason why bootstrap is the first stuff to be deleted/unbind from nydusd